sass/compat/themes/minimog/_checkout-payment-methods.scss

```diff
@@ -0,0 +1,7 @@
+// ------------------------------------
+// CHECKOUT PAYMENT METHOD
+// ------------------------------------
+
+body#{ $_body-theme-selector } .wc_payment_methods li+li{
+    margin: 0!important;
```

Comment:
Add space between CSS property value and the `!important` directive.
Whenever using `!important`, a comment explaining why it was used is required and the end of the line, that is after the `;`.